docs/content/en/integrations/import.md

```diff
@@ -461,6 +461,29 @@ Example of JSON format:
 }
 ```
 
+This parser support an attributes that accept files as Base64 strings. These files will be created as `UploadFile`
```

Comment:
Where does the term `UploadFile` come from? In the UI they are referenced as "Files". 

Suggestion:

```suggestion
This parser support an attributes that accept files as Base64 strings. These files are attached to the respective findings.
```